### PR TITLE
NEW PR

### PR DESCRIPTION
**Pull request by haradakunihiko**
@kunihikoharadatest

### CODE REVIEW

**Comment by haradakunihiko** [comment]
@kunihikoharadatest 

asflk

File: test.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+console.log('aa')
+
```
Comment (haradakunihiko) [inline test.js:2]: PRナイノコメント

Comment (haradakunihiko) [reply test.js:2]: @kunihikoharadatest 

asfa

File: test.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
```
Comment (haradakunihiko) [inline test.js:3]: @kunihikoharadatest 
PRにコメントするよ

File: test.js (hunk 1 above)
Comment (haradakunihiko) [inline test.js:2]: @kunihikoharadatest 
hoge

Comment (haradakunihiko) [reply test.js:2]: @kunihikoharadatest 
hage

File: test.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+console.log('aa')
```
Comment (haradakunihiko) [inline test.js:1]: @kunihikoharadatest 
huga

Comment (haradakunihiko) [reply test.js:1]: @kunihikoharadatest 
hoho

Comment (haradakunihiko) [reply test.js:1]: @kunihikoharadatest 
in a review comment

**Review by haradakunihiko — COMMENTED** [review]
@kunihikoharadatest

File: test.js (hunk 1 above)
Comment (haradakunihiko) [inline test.js:2]: start review 
@kunihikoharadatest

Comment (haradakunihiko) [reply test.js:2]: started review
@kunihikoharadatest

**Review by haradakunihiko — COMMENTED** [review]
@kunihikoharadatest 
review confirm

**Comment by kunihikoharadatest** [comment]
comment in PR

**Comment by kunihikoharadatest** [comment]
LGTM

File: test.js (hunk 3 above)
Comment (kunihikoharadatest) [inline test.js]: うーん

Comment (kunihikoharadatest) [reply test.js]: どうかな

**Pull request closed without merging**